plotly/colors.py

```diff
@@ -339,7 +360,30 @@ def convert_dict_colors_to_same_type(colors, colortype='rgb'):
                                      "for your colortype variable.")
 
 
-def make_colorscale(colors, scale=None):
+def validate_scale_values(scale):
+    """
+    Validates scale values from a colorscale
+    """
+    if len(scale) < 2:
+        raise exceptions.PlotlyError("You must input a list of scale values "
+                                     "that has at least two values.")
```

Comment:
🐄 as a heads up, we typically only use `"` when there is a `'` character in the string. Otherwise, we use `'`, whenever possible